legacy/reqcounter/reqcounter.go

```diff
@@ -29,10 +29,11 @@ import (
 // TODO: @tylerferrara in the future, add a field 'persistent bool' to determine if
 // the request counter should ever reset.
 type RequestCounter struct {
-	Mutex    sync.Mutex    // Lock to prevent race-conditions with concurrent processes.
-	Requests uint64        // Number of HTTP requests since recording started.
-	Since    time.Time     // When the current request counter began recording requests.
-	Interval time.Duration // The duration of time between each log.
+	Mutex      sync.Mutex    // Lock to prevent race-conditions with concurrent processes.
+	Requests   uint64        // Number of HTTP requests since recording started.
+	Since      time.Time     // When the current request counter began recording requests.
+	Interval   time.Duration // The duration of time between each log.
+	Persistent bool          // Determine if the request counter should ever reset.
```

Comment:
Nit: I think the comment is easier to read if you just tell the reader what will happen. E.g.,

```
// If true, the Requests field is never set back to 0 when reset() is called.
```

Reply:
Actually, maybe `Resettable` is a better name? It's more to the point and is almost self-explanatory.

Reply:
Ack.